app/assets/javascripts/host_edit.js

```diff
@@ -762,6 +762,11 @@ function disable_vm_form_fields() {
   $("[id^=host_compute_attributes]").each(function () {
     $(this).attr("disabled", "disabled");
   });
+  $("[id^=host_interfaces_attributes]").filter(function() {
+    return this.id.match(/^host_interfaces_attributes_[0-9]+_compute_attributes_.*/);
+  }).each(function () {
+    $(this).attr("disabled", "disabled");
```

Comment:
I agree, however we should prioritize. Imho fixing the issue is more important. We can then refactor the whole code to disable _all_ fields via views.